### DISCUSSION TITLE
SECURITY: Two-Factor Authentication

### DISCUSSION THREAD

**Discussion by ACPK** [plausible/analytics#1380] [Anything Else]
It would be great if the site had two-factor (multi-factor) authentication because security should be important.

**Comment by Vigasaurus** [comment]
#156 has more context

**Comment by metmarkosaric** [comment]
**Update:** You can now enable two-factor authentication (2FA) as an extra security layer for your Plausible account in your account settings. Here’s how: https://plausible.io/docs/2fa
